src/components/Core/Notifications/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './notification.module.css';
+
+export interface NotificationType {
+    message: string
+    success: boolean
+}
+
+export const Notification = ({message, success}: NotificationType) => {
+    
+    const inlineStyles = success ? {
+        backgroundColor: "#e0ffe3",
+        color: "#00780c"
+    } : {
+        backgroundColor: "#ffd9d9",
+        color: "#9c0000"
+    };
+    
+    return (
+        <div style={inlineStyles} className={styles.notification}>
```

Comment:
Perhaps we can just use an alert from MUI? Then we don't have to add any own CSS classes? Or perhaps a snackbar? Snackbars might be too "far" from the form however

https://mui.com/components/alert/
https://mui.com/components/snackbars/

Reply:
I feel alert is better